packages/babel-helpers/src/helpers/using.js

```diff
@@ -0,0 +1,19 @@
+/* @minVersion 7.0.0-beta.0 */
+
+export default function _using(stack, value, isAwait) {
+  if (value !== null && value !== void 0) {
+    // core-js-pure uses Symbol.for for polyfilling well-known symbols
+    if (isAwait) {
+      var dispose =
+        value[Symbol.asyncDispose || Symbol.for("Symbol.asyncDispose")];
```

Comment:
One thought: what about the performance of `Symbol.for`? Maybe we can cache it? I guess the `using` proposal might be popular in the future.

Reply:
Isn't `Symbol.for` _very fast_? It's wrapping a string in a (C++) object, without doing any computation.

Reply:
You are right! This should be fast!